cosmwasm/submit-proposal.js

```diff
@@ -176,11 +176,11 @@ const registerItsChain = async (client, wallet, config, options) => {
         const chainConfig = getChainConfig(config, chain);
         const { maxUintBits, maxDecimalsWhenTruncating } = getChainTruncationParams(config, chainConfig);
 
-        const itsEdgeContract = getItsEdgeContract(chainConfig);
+        const itsEdgeContractArg = itsEdgeContract(chainConfig);
 
         return {
             chain: chainConfig.axelarId,
-            its_edge_contract: itsEdgeContract,
+            its_edge_contract: itsEdgeContractArg,
```

Comment:
```suggestion
            its_edge_contract: itsEdgeContract(chainConfig),
```
avoid creating intermediate vars when not needed. A bad var name usually indicates this issue